src/firefly/java/edu/caltech/ipac/visualize/plot/plotdata/Wavelength.java

```diff
@@ -561,4 +561,24 @@ else if (sArray[1].trim().equalsIgnoreCase("LOG")){
         double[] p_j={p0, p1, p2};
         return p_j;
     }
+
+    /* Lijun Zhang 11/07/19
+     * This main is added to test if the FitsTableReader can read the table with array correctly
+     * In the WAVE-TAB case:
+     * The  look up table contains only one row, one or two columns.  The coordinates column and an optional index
+     * vector column.
+     * The data stored as an array in each cell.  The array size in coordinate and index can be different.
+     *
+     * Using FitsTableReader, the array in the cell is flatten and the result table is the multi-row with single value
+     * at each cell.  The length of rows equals to the maximum array length between two columns. There fore the
+     * table processed is two column and muliti-rows table.
+     *
+     *
+    */
+    public static void main(String[] args) {
+
+    }
```

Comment:
I think a unit test should be better instead

Reply:
Sorry, I don't mean to tag this one.  The java version is already deprecated.

Reply:
It will be good to add a unit test.  @ejoliet  you can issue a ticket to add a unit test for wavelength.  It will take some time to get it done since it has many cases to be tested.

Reply:
@ejoliet  I reverted the Wavelength.java to its dev's version.  That block you saw is not a test.  I was trying to find out if the array data in the FITs read OK.  Later I learned the table is accessible in the front end.  But I forgot to delete this empty main function.  FYI